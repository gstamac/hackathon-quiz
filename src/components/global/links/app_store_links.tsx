import React from 'react'
import { useStyles } from './styles'
import { APPLE_STORE_LINK, GOOGLE_STORE_LINK } from '../../../constants'
import iosStoreIcon from '../../../assets/icons/ios_store.svg'
import googlePlayStore from '../../../assets/icons/google_play_store.svg'
import { getString } from '../../../utils'

export const AppStoreLinks: React.FC = () => {
  const { getTheAppLinks } = useStyles()

  return <div className={getTheAppLinks}>
    <a href={APPLE_STORE_LINK} target='_blank' rel='noopener noreferrer'>
      <img src={iosStoreIcon} alt={getString('ios-app-store')} />
    </a>
    <a href={GOOGLE_STORE_LINK} target='_blank' rel='noopener noreferrer'>
      <img src={googlePlayStore} alt={getString('google-play-store')} />
    </a>
  </div>
}
