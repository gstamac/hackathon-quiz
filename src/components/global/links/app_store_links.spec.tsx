import React from 'react'
import { render, RenderResult } from '../../../../tests/test_utils'
import { AppStoreLinks } from './app_store_links'

describe('AppStoreLinks', () => {
  it('should render links to app store', () => {
    const renderResult: RenderResult = render(<AppStoreLinks />)

    expect(renderResult.container).toMatchSnapshot()
  })
})
