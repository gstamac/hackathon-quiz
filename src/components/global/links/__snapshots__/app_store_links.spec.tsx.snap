// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppStoreLinks should render links to app store 1`] = `
<div>
  <div
    class="makeStyles-getTheAppLinks-1"
  >
    <a
      href="https://apps.apple.com/us/app/globalid-portable-identity/id1439340119"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Download app from Apple app store"
        src="ios_store.svg"
      />
    </a>
    <a
      href="https://play.google.com/store/apps/details?id=net.globalid"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="Download app from Google App store"
        src="google_play_store.svg"
      />
    </a>
  </div>
</div>
`;
